docs/spec.md

```diff
@@ -176,6 +181,60 @@ Informs the client about parameters. Only supported if the server declares the `
 }
 ```
 
+### Advertise Services
+
+Informs the client about available services. Only supported if the server declares the `services` [capability](#server-info).
+
+#### Fields
+
+- `op`: string `"advertiseServices"`
+- `services`: array of:
+  - `id`: number. The server may reuse ids when services disappear and reappear, but only if the services keeps the exact same name, type, and schema. Clients will use this unique id to cache schema info and deserialization routines.
+  - `name`: string
+  - `type`: string
+  - `schema`: string
+  - `acceptedEncodings`: array of
+    - `id`: number
+    - `name`: string, name of an encoding which can be used to encode service call requests
```

Comment:
We could also just make that a string and allow only a single request encoding?

Reply:
I like the array, but why the `id` field?